tests/zip/zip_test_lib.py

```diff
@@ -91,4 +91,5 @@ def assertZipFileContent(self, zip_file, content):
             # legacy rule implementation.
             attr = 0o555
           self.assertEqual(oct((info.external_attr >> 16) & UNIX_RWX_BITS),
-                           oct(attr))
+                           oct(attr),
+                           info.filename)
```

Comment:
Nit: Maybe call out the keyword arg `msg` here?

Reply:
Done